themes/bootstrap3/js/combined-search.js

```diff
@@ -4,8 +4,9 @@ VuFind.combinedSearch = (function CombinedSearch() {
   function init(container, url) {
     VuFind.loadHtml(container, url, '', function containerLoad(responseText) {
       if (responseText.length === 0) {
-        container.hide();
-        let parent = container.parent();
+        const $container = $(container);
+        $container.hide();
+        let parent = $container.parent();
```

Comment:
I used jQuery here originally only because it was [already being passed](https://github.com/vufind-org/vufind/pull/3233#discussion_r1402068771) in as a jQuery element, which you've fixed in this PR.  So you could replace this jQuery as well.  [This site](https://youmightnotneedjquery.com/) has one-line HTML replacements for hide(), parent(), and hasClass().